### PR TITLE
Feature/form create zone calendar

### PR DESCRIPTION
**Pull request by MilanMilanMilan**

### CODE REVIEW

File: public/calendar/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (JanVan01) [inline public/calendar/index.html]: Do we need this? Is all the calendar stuff necessary/used?

File: public/css/dist/leaflet.draw.css [hunk 2]
```diff
@@ -0,0 +1,305 @@
+/* ================================================================== */
+/* Toolbars
```
Comment (JanVan01) [inline public/css/dist/leaflet.draw.css]: Do we need this? Or is it loaded over a CDN?

File: public/index.html [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+		<meta charset="utf-8"></meta>
+		<!-- style imports -->
+		<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+		<link href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" rel="stylesheet">
+		<link href="/css/style.css" rel="stylesheet">
+		<link href="/css/dist/leaflet.draw.css" rel="stylesheet">
+		<link href='https://api.mapbox.com/mapbox.js/v2.4.0/mapbox.css' rel='stylesheet' />
+		<link rel="stylesheet" href="http://leaflet.github.io/Leaflet.draw/lib/leaflet/leaflet.css"/>
+        <link rel="stylesheet" href="http://leaflet.github.io/Leaflet.draw/leaflet.draw.css"/>
+		<link rel="stylesheet" href="https://raw.githubusercontent.com/CliffCloud/Leaflet.EasyButton/master/src/easy-button.css"/>
```
Comment (JanVan01) [inline public/index.html]: Remove that

File: public/index.html [hunk 4]
```diff
@@ -1,28 +1,38 @@
...
+		<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+		<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+		<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vue/1.0.22/vue.min.js"></script>
+		<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vue-resource/0.7.0/vue-resource.min.js"></script>
+		<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
+		<script type="text/javascript" src='https://api.mapbox.com/mapbox.js/v2.4.0/mapbox.js'></script>
+		<script type="text/javascript" src="http://leaflet.github.io/Leaflet.draw/leaflet.draw.js"></script>
+		<script type="text/javascript" src="http://leaflet.github.io/Leaflet.draw/lib/leaflet/leaflet.js"></script>
+		<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/0.2.3/leaflet.draw-src.js"></script>
+		<script type="text/javascript" src="https://raw.githubusercontent.com/CliffCloud/Leaflet.EasyButton/master/src/easy-button.js"></script>
```
Comment (JanVan01) [inline public/index.html]: That as well

File: public/index.html [hunk 5]
```diff
@@ -1,28 +1,38 @@
...
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.6/Chart.bundle.min.js"></script>
     <!-- Did not find any CDN -->
     <script type="text/javascript" src="../source/leaflet-pip.min.js"></script>
     <script type="text/javascript" src="../source/geojson.min.js"></script>
-    
 
-</head>
 
-<body>
+	</head>
```
Comment (JanVan01) [inline public/index.html]: Indentation

File: public/js/CalenderPopup.js [hunk 6]
```diff
@@ -0,0 +1,1465 @@
+// ===================================================================
```
Comment (JanVan01) [inline public/js/CalenderPopup.js]: needed?

File: public/js/addZoneVue.js [hunk 7]
```diff
@@ -1,17 +1,57 @@
...
+			toFloat = parseFloat(this.coordinatesmodel);
+			parsedjson.Geometry.Coordinates.push(toFloat);
+			console.log(toFloat),
+		//	console.log(parsedjson.Geometry.Coordinates);
+			console.log(JSON.stringify(parsedjson));
+		},
+		submitzone: function () {
+			jsonstring = JSON.stringify(vmAddZone.$data);
+			console.log(jsonstring);
+//	$.post("/api/addzone",jsonstring);
```
Comment (JanVan01) [inline public/js/addZoneVue.js]: Do you need this or can it be deleted?

File: public/js/addZoneVue.js [hunk 8]
```diff
@@ -1,17 +1,57 @@
...
-        }
-    }
+var vmAddZone = new Vue({
+	el: '#addzone',
+	parent: vue_broadcaster,
+	data: {
+		state: false,
+		Geometry: {
+			Type: "Polygon",
+			Coordinates: []//[52.00094684771797,7.474822998046875],[51.94511140570431,7.3828125],[51.90869633027845,7.466583251953125],[52.00094684771797,7.474822998046875]
```
Comment (JanVan01) [inline public/js/addZoneVue.js]: Remove

File: public/js/buttonVue.js [hunk 9]
```diff
@@ -1,4 +1,4 @@
-var vm = new Vue({
+var vmButton = new Vue({
```
Comment (JanVan01) [inline public/js/buttonVue.js]: remove the name completely

File: public/calendar/index.html (hunk 1 above)
Comment (schokocookie25) [inline public/calendar/index.html]: if we can use the bootstrap datepicker, we won't need the calender stuff anymore.

**Comment by Flugmango** [comment]
I don't know, this looks like there is so much stuff coming just for the calendar, can't we just use something like this? https://jqueryui.com/datepicker/

**Comment by schokocookie25** [comment]
I think it is simiar to the bootstrap datepicker. I tried that one but the source code is not available. If you can find the code we can also use it.

File: public/js/addZoneVue.js (hunk 7 above)
Comment (MilanMilanMilan) [inline public/js/addZoneVue.js]: deleted

File: public/index.html (hunk 3 above)
Comment (MilanMilanMilan) [inline public/index.html]: ok

File: public/index.html (hunk 4 above)
Comment (MilanMilanMilan) [inline public/index.html]: ok

File: public/index.html (hunk 5 above)
Comment (MilanMilanMilan) [inline public/index.html]: fixed

File: public/js/addZoneVue.js (hunk 8 above)
Comment (MilanMilanMilan) [inline public/js/addZoneVue.js]: ok

File: public/css/dist/leaflet.draw.css (hunk 2 above)
Comment (MilanMilanMilan) [inline public/css/dist/leaflet.draw.css]: I'm not sure

**Pull request merged**